scripts/process_schemas.py

```diff
@@ -343,20 +359,31 @@ def run(self):
         sg = SchemaGenerator(self.version, directory, self.destinationFile)
         sg.write()
 
+    def run(self):
+        if self.sourceTar is None:
+            self.runWithDir(self.schemaDir)  # local mode
+        else:
+            self.runWithGitHub()
+
 
 def main():
     parser = argparse.ArgumentParser(
         description="Script to process GA4GH Avro schemas. Requires "
         "java external command")
     parser.add_argument(
-        "--outputFile", "-o", default="ga4gh/_protocol_definitions.py",
+        "--output-file", "-o", default="ga4gh/_protocol_definitions.py",
```

Comment:
There is inconsistency in some of the command line args all right @Naburimannu. Perhaps we should move to making all long-form arguments camelCase?

But yes, I very much support @Naburimannu's camelCasing comments here.